Apostrophe in shortcut name field breaking packaging.

**Describe the bug**
One of our users over at PWABuilder was attempting to package their site: https://www.lagendanews.com/. In their manifest they have inside of their shortcuts field `"name": "L'Agenda N"` however, in the bubblewrap service this becomes `name:'L'Agenda N'` which is breaking due to the single tick inside the name string.

Should we advise them to not using ticks in their string or is this a bug?

**To Reproduce**
Steps to reproduce the behavior:
You can use the bubblewrap cli with the manifest url https://www.lagendanews.com/superpwa-manifest.json with default options, should fail at the `bubblewrap build` step

**Expected behavior**
Expected this character to be escaped or something like that in order to allow packaging

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Edge
 - Version: Current
 
**Additional context**
Associated PWABuilder issue: https://github.com/pwa-builder/PWABuilder/issues/3817

I can repro the issue. Here's the relevant bit of the issue:
```
Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
build file 'D:\Projects\twa-project\app\build.gradle': 41: unexpected token: Agenda @ line 41, column 26.
       shortcuts: [[name:'L'Agenda N', short_name:'L'Agenda N', url:'https://www.lagendanews.com/', icon:'shortcut_0']],
```

The issue is in [`ShortcutInfro#toString()`](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/src/lib/ShortcutInfo.ts#L49-L52), which doesn't properly escape the string. The string can be escaped with the [`escapeGradleString()` method](https://github.com/GoogleChromeLabs/bubblewrap/blob/24ec1acfd8beda03d3d7535943351846da45a6a2/packages/core/src/lib/util.ts#L322-L324).

--- a/packages/core/src/lib/ShortcutInfo.ts
+++ b/packages/core/src/lib/ShortcutInfo.ts
@@ -16,7 +16,7 @@
 
 'use strict';
 
-import {findSuitableIcon} from './util';
+import {findSuitableIcon, escapeGradleString} from './util';
 import {WebManifestShortcutJson, WebManifestIcon} from './types/WebManifest';
 
 // As described on https://developer.chrome.com/apps/manifest/name#short_name
@@ -47,8 +47,10 @@ export class ShortcutInfo {
   }
 
   toString(index: number): string {
-    return `[name:'${this.name}', short_name:'${this.shortName}', ` +
-      `url:'${this.url}', icon:'${this.assetName(index)}']`;
+    return `[name:'${escapeGradleString(this.name)}', ` +
+      `short_name:'${escapeGradleString(this.shortName)}', ` +
+      `url:'${this.url}', ` +
+      `icon:'${this.assetName(index)}']`;
   }
 
   assetName(index: number): string {

